tests/std/tests/VSO_1062649_overaligned_function/test.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+
+#ifndef __cpp_aligned_new
+#pragma error("This test is only valid when C++17 over-aligned allication is supported")
+#endif
+
+#pragma warning(disable : 4324) // "was padded due to alignment specifier", thanks, Captain
+
+struct alignas(16) overaligned_t {
+    char non_empty;
+
+    void operator()(const void* storage, std::size_t storage_size) const {

```

Comment:
We usually `const` everything under the sun:
```suggestion
    void operator()(const void* const storage, const std::size_t storage_size) const {
```